### PR TITLE
Fix homepage anime slider

### PR DESCRIPTION
**Pull request by tokitouq**

### CODE REVIEW

File: backend/components/scroll_area/scroll_area.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <scroll-area
+    {% if hyperscript %}_="{{ hyperscript }}"{% endif %}
```
Comment (baseplate-admin) [inline backend/components/scroll_area/scroll_area.html:2]: Remove this

**Review by baseplate-admin — COMMENTED** [review]
image

File: backend/django_core/templates/anime/_partial.html [hunk 2]
```diff
@@ -65,26 +63,22 @@
...
-                    <button class="btn btn-primary p-0 absolute -right-[1vw] top-[12vw] z-20 hidden h-[2.25vw] w-[2.25vw] rounded-[0.375vw] bg-secondary-800 md:flex">
-                        <svg xmlns="http://www.w3.org/2000/svg" class="w-[1.25vw] -rotate-90" fill="none" viewBox="0 0 24 24">
-                            <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="3.5" d="M6 9L12 15L18 9"></path>
-                        </svg>
+                    <button class="btn btn-primary p-0 absolute -right-[1vw] top-[12vw] z-20 hidden h-[2.25vw] w-[2.25vw] rounded-[0.375vw] bg-secondary-800 md:flex"
+                    >
+                        {% include "icons/chevron.html" with class="w-[1.25vw] -rotate-90" %}
                     </button>
                 </latest-animes-slider>
                 <latest-episodes class="hidden w-[21.5625vw] md:block">
```
Comment (baseplate-admin) [inline backend/django_core/templates/anime/_partial.html:84]: image

File: backend/django_core/static/vendor/js/swiper/swiper.min.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+//# sourceMappingURL=swiper.min.js.map
```
Comment (baseplate-admin) [inline backend/django_core/static/vendor/js/swiper/swiper.min.js]: Add map files

File: backend/django_core/templates/tailwind_base.html [hunk 4]
```diff
@@ -28,6 +28,7 @@
         />
         <script src="{% static 'vendor/js/htmx/htmx.min.js'%}"></script>
         <script src="{% static 'vendor/js/hyperscript/hyperscript.min.js'%}"></script>
+        <script src="{% static 'vendor/js/swiper/swiper.min.js'%}"></script>
```
Comment (baseplate-admin) [inline backend/django_core/templates/tailwind_base.html:34]: Importing something like swiper at this level is not a good practice

File: backend/django_core/templates/anime/home.html [hunk 5]
```diff
@@ -196,7 +196,6 @@
 {% endblock %}
 
 {%block js %}
-    <script src="https://cdn.jsdelivr.net/npm/swiper@11/swiper-bundle.min.js"></script>
     {% component_js_dependencies %}
     <script>
         const swiper = new Swiper('.swiper', {
```
Comment (baseplate-admin) [inline backend/django_core/templates/anime/home.html:197]: Add anonymous functions here. Read more about [this](https://baseplate-admin.github.io/blog/how-i-fixed-htmx-redeclaration-of-let-element-with-hx-boosttrue.html)

File: backend/django_core/templates/tailwind_base.html (hunk 4 above)
Comment (tokitouq) [reply backend/django_core/templates/tailwind_base.html:34]: > Importing something like swiper at this level is not a good practice

Then? import inside `_partial.html`?

Comment (baseplate-admin) [reply backend/django_core/templates/tailwind_base.html:34]: > Then? import inside _partial.html?

Correct, import the function where you declare swiper class

File: backend/django_core/templates/user/register/_1.html [hunk 6]
```diff
@@ -1,35 +1,38 @@
...
-            ...(await import('https://cdn.jsdelivr.net/npm/@zxcvbn-ts/language-ar@latest/+esm')).dictionary,
-        },
-    }
-    zxcvbnTscore.zxcvbnOptions.setOptions(options)
-    function get_password_strength(password){
-        return zxcvbnTscore.zxcvbn(password)
-    }
-    window.get_password_strength = get_password_strength
+    (async()=>{
+        if(!navigator.onLine){return};
```
Comment (baseplate-admin) [inline backend/django_core/templates/user/register/_1.html:5]: @tokitouq, add this line. Otherwise the function doesn't get executed

**Pull request merged**